app/views/includes/page-title.html

```diff
@@ -0,0 +1,3 @@
+{% if subTitle %}{{ subTitle }} - {% endif %}
+{% if title %}{{ title }}{% endif %} 
+- WMT
```

Comment:
Maybe include the `-` in the second `if`. Then if there is no subtitle or title the heading won't be `- WMT`, but `WMT`?

Reply:
Good spot, fixed